.github/build-iroha1.src.yml

```diff
@@ -492,6 +490,7 @@ jobs:
           -GNinja
           ${{ matrix.CMAKE_USE }}
           -DTESTING=ON
+          -DBENCHMARKING=ON
```

Comment:
you sure we need that?

Reply:
No, but @iceseer complied that we must be able to compile them, so lets check. On Jenkins this flag was enabled.

I believe the right name of this flag is `ENABLE_BECHMARKING_TESTS`